src/geovista/core.py

```diff
@@ -69,18 +66,9 @@
     "slice_mesh",
 ]
 
-CUT_EAST: str = "EAST"
-"""Preference for a slice to bias cells east of the chosen meridian."""
-
-CUT_EXACT: str = "EXACT"
-"""Preference for a slice to be true to the chosen meridian."""
-
 CUT_OFFSET: float = 1e-5
 """Cartesian west/east bias offset of a slice."""
 
-CUT_WEST: str = "WEST"
-"""Preference for a slice to bias cells west of the chosen meridian."""
-
```

Comment:
I found the relationship between these and `SliceBias` quite difficult to understand. Everything seems to work just as well if I make `SliceBias` the single common language throughout the codebase.

Reply:
Originally I was unsure about forcing users to use `Enum`s ... I'm hummed and hawed about it then moved on.

Glad you've picked up on it and forced a decision 👍

Reply:
Yes I've wondered about it myself!